packages/app-root/src/app/page.js

```diff
@@ -14,8 +14,8 @@ function parseFeedPost(post) {
     title: post.title, // string
     excerpt: post.excerpt, // string but text is wrapped in <p></p>
     created_at: new Date(post.date), // string such as '2024-02-02T15:00:00+00:00'
-    link: post.URL, // string
-    image: post.featured_image // ?? these are always '', might need to grab attached image instead
+    url: post.URL, // string
+    imageSrc: post.featured_image // ?? these are always '', might need to grab attached image instead
```

Comment:
I'll double check with Sean if he prefers a placeholder image for blog posts that don't have a `featured_image`. All of the most recent blogs posts do not include one, as seen in the blog section of PFE's homepage: https://www.zooniverse.org